src/rpc/server.cpp

```diff
@@ -281,9 +276,9 @@ UniValue help(const JSONRPCRequest& jsonRequest)
         );
 
     std::string strCommand, strSubCommand;
-    if (jsonRequest.params.size() > 0)
+    if (!jsonRequest.params.size() > 0)
```

Comment:
smth went wrong here - should be no `!`
```suggestion
    if (jsonRequest.params.size() > 0)
```